### DISCUSSION TITLE
Any questions please raise in this thread

### DISCUSSION THREAD

**Discussion by IoT-Hub-RU** [java-embedded-framework/jef#2] [Q&A] [answered]
Please raise any questions here

**Comment by nOy39** [comment, marked as answer]
Hi, thx you for your big job. Could you check your Github repository. I have 

> Cannot resolve ru.iothub.jef:linux-core:0.0.1-ALPHA
message when try install dependencies from your manual.

**Reply by IoT-Hub-RU** [reply]
Hi Andrey,
did you add Github to repos to pom.xml as described in README.md?
At this moment libs not published to maven central and can be downloaded from maven-github.

I think you need to add this section to pom.xml file and

```
 <distributionManagement>
        <repository>
            <id>jef</id>
            <name>GitHub Java Embedded Frameworks Packages</name>
            <url>https://maven.pkg.github.com/java-embedded-framework/jef</url>
            <uniqueVersion>false</uniqueVersion>
        </repository>
    </distributionManagement> 
```


 add github to settings.xml like here:
https://github.community/t/maven-build-with-two-github-packages-repositories-fails/139014/2

or download jar's and install them locally like here - https://mkyong.com/maven/how-to-include-library-manully-into-maven-local-repository/

or other option:
clone or download project and build/install to local repo manually.
